### PR TITLE
logs:improved crash logs during failure with keploy

### PR DESCRIPTION
**Pull request by gouravkrosx**
Related issue: #45 

Signed-off-by: gouravkrosx <gouravgreatkr@gmail.com>

### CODE REVIEW

**Review by Sarthak160 — APPROVED** [review]

File: api/src/main/java/io/keploy/service/GrpcService.java [hunk 1]
```diff
@@ -279,46 +282,68 @@ public static void Test() {
...
         logger.debug("inside start function");
         Service.startRequest startRequest = Service.startRequest.newBuilder().setApp(k.getCfg().getApp().getName()).setTotal(total).build();
-        Service.startResponse startResponse = blockingStub.start(startRequest);
-        return startResponse.getId();
+        Service.startResponse startResponse = null;
+
+        try {
+            startResponse = blockingStub.start(startRequest);
+        } catch (Exception e) {
+            logger.error("failed to start test run, please check keploy logs", e);
```
Comment (slayerjain) [inline api/src/main/java/io/keploy/service/GrpcService.java:290]: this should be `keploy server logs` for added clarity.

File: api/src/main/java/io/keploy/service/GrpcService.java [hunk 2]
```diff
@@ -328,11 +353,12 @@ public static List<Service.TestCase> fetch() {
     }
 
     public static boolean check(String testrunId, Service.TestCase tc) {
-        logger.debug("running test case with [{}] testrunId ", testrunId);
+        logger.debug("running test case with [{}] testrunId", testrunId);
 
         Service.HttpResp resp;
         try {
             resp = simulate(tc);
+            logger.debug("response getting from simulate request: {}", resp);
```
Comment (slayerjain) [inline api/src/main/java/io/keploy/service/GrpcService.java:361]: `response got from simulate` here

**Review by slayerjain — CHANGES_REQUESTED** [review]
mostly LGTM

**Review by gouravkrosx — COMMENTED** [review]
Resolved the mentioned changes

**Pull request merged**